freight/models/app.py

```diff
@@ -82,3 +82,36 @@ def get_default_ref(self, env):
         if not data:
             return DEFAULT_REF
         return data.get('default_ref', DEFAULT_REF)
+
+    def get_current_sha(self, env):
```

Comment:
Yeah I agree re: the code in the views. I've never found a happy pattern for where this kind of code should live. It generally ends up on models or in some random lib folder